fe-library-app/src/components/SearchBar/SearchBar.tsx

```diff
@@ -1,33 +1,53 @@
 import { useState } from 'react'
 
+import { Filters } from '../AppRouter/pages/HomePage'
 import Filter from '../Filter/Filter'
+import Order from '../Order/Order'
 import Search from '../Search/Search'
 import style from './SearchBar.module.css'
 
 interface SearchBarProps{
   searchChange: (newInput: string) => void
+  filterChange: (filterData: Filters) => void
+  orderChange: (orderData: string[]) => void
 }
 
 const SearchBar = (props: SearchBarProps) => {
   const [ showFilter, setShowFilter ] = useState(false)
+  const [ showOrder, setShowOrder ] = useState(false)
 
-  const onFilterHandler = () => {
+  const clickFilterHandler = () => {
```

Comment:
One more blog for reading: https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
If you are sometimes confused what is the best way to give a name to a function, there are some suggestion for best practices.

Reply:
Thanks